packages/hooks/use-theme/src/index.ts

```
@@ -0,0 +1,93 @@
+
+    try {
+      storedTheme = localStorage.getItem(ThemeProps.KEY) || undefined;
+    } catch {}
+
+    // return stored theme if it is selected previously
+    if (storedTheme) return storedTheme;
+
+    // if it is using system theme, check `prefers-color-scheme` value
+    // return light theme if not specified
+    if (defaultTheme === ThemeProps.SYSTEM) {
+      return window.matchMedia && window.matchMedia(MEDIA).matches
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)